fenix/app/metrics.yaml

```diff
@@ -10850,6 +10850,25 @@ fx_suggest:
     expires: never
     send_in_pings:
       - fx-suggest
+  position:
+    type: quantity
+    unit: non-negative integer
+    description: >
+      The position (1-based) of this suggestion in the full list of suggestions, relative to the
+      top of the awesomebar.
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1858542
+    data_reviews:
+      - TODO
+    data_sensitivity:
+      - interaction
+    notification_emails:
```

Comment:
We should also be sending notification emails to "android-probes@mozilla.com".

Reply:
Thanks, added that group to all the `fx_suggest` metrics!